TheMovieDb/Home/Utils/MovieDummy.swift

```diff
@@ -0,0 +1,31 @@
+        
+        let movie2 = Movie(id: 1, title: "The batman", video: false, voteAverage: 20, overview: "The Lord of the Rings: The Fellowship of the Ring. Young hobbit Frodo Baggins, after inheriting a mysterious ring from his uncle Bilbo, must leave his home in order to keep it from falling into the hands of its evil creator. Along the way, a fellowship is formed to protect the ringbearer and make sure that the ring arrives at its final destination: Mt. Doom, the only place where it can be destroyed.", releaseDate: "2001-12-18", voteCount: 345, adult: false, backdropPath: "/y9AuabF1SXnn3xZ0tAJLLhv33Uj.jpg", posterPath: "/8RW2runSEc34IwKN2D1aPcJd2UL.jpg", popularity: 51.198, mediaType: "movie")
+        
+        let movie3 = Movie(id: 1, title: "The batman", video: false, voteAverage: 20, overview: "Following the death of District Attorney Harvey Dent, Batman assumes responsibility for Dent's crimes to protect the late attorney's reputation and is subsequently hunted by the Gotham City Police Department. Eight years later, Batman encounters the mysterious Selina Kyle and the villainous Bane, a new terrorist leader who overwhelms Gotham's finest. The Dark Knight resurfaces to protect a city that has branded him an enemy.", releaseDate: "2001-12-18", voteCount: 345, adult: false, backdropPath: "/bsg0mrxUKyJoL4oSGP5mlhEsqp.jpg", posterPath: "/rpOqHZMNIaI4qP4r7nw1B7oA0mx.jpg", popularity: 51.198, mediaType: "movie")
+        
+        let movie4 = Movie(id: 1, title: "The batman", video: false, voteAverage: 20, overview: "Cobb, a skilled thief who commits corporate espionage by infiltrating the subconscious of his targets is offered a chance to regain his old life as payment for a task considered to be impossible: \"inception\", the implantation of another person's idea into a target's subconscious.", releaseDate: "2001-12-18", voteCount: 345, adult: false, backdropPath: "/pcDc2WJAYGJTTvRSEIpRZwM3Ola.jpg", posterPath: "/tnAuB8q5vv7Ax9UAEje5Xi4BXik.jpg", popularity: 30, mediaType: "movie")
+        
+        let movie5 = Movie(id: 1, title: "The batman", video: false, voteAverage: 20, overview: "hey", releaseDate: "2001-12-18", voteCount: 345, adult: false, backdropPath: "/pcDc2WJAYGJTTvRSEIpRZwM3Ola.jpg", posterPath: "/tnAuB8q5vv7Ax9UAEje5Xi4BXik.jpg", popularity: 51.198, mediaType: "movie")
+        movies.append(movie1)
+        movies.append(movie2)
+        movies.append(movie3)
+        movies.append(movie4)
+        movies.append(movie5)
+        return movies
```

Comment:
why are we hardcoding this values?

Reply:
It was my dummy data for render in view, I already deleted  It